pagai/__init__.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from dotenv import load_dotenv
+from flask import Flask
+from pagai.views import api
+
+app = Flask(__name__)
+app.register_blueprint(api)
+
+
+if __name__ == "__main__":
+    # Load .env config file for entire environement
+    configFileName = (
+        ".env.dev.custom" if os.path.exists(".env.dev.custom") else ".env.dev.default"
```

Comment:
can you make this configurable ? eg: have an env var like CONFIG_PATH (already talked about this with @elsiehoffet-94 )

Reply:
For example: `configFileName = os.getenv("CONFIG_PATH",'.env.dev.default')`